### PR TITLE
Data mgmt

### PR DESCRIPTION
**Pull request by datailluminations**
Please review the latest changes in Data Management page:

The DataLoader(parent component) successfully retrieve the list of datasets from our backend via API call.

It passes the array of JSON objects containing the list of datasets from the backend to the CheckboxParent which renders the list of datasets and their associated checkboxes. It also remembers which boxes are check.  So data passing between parent and child component all works!

### CODE REVIEW

File: springdb/data [hunk 1]
```diff
@@ -0,0 +1 @@
+{[{"dataset":"Targets", "description":"Core annotation for targets", "source":"ftp://ftp.ebi.ac.uk/pub/databases/opentargets/platform/latest/output/etl/parquet/targets", "timestamp":1647831895},{"dataset":"Drugs", "description":"Core annotation for drugs", "source":"ftp://ftp.ebi.ac.uk/pub/databases/opentargets/platform/latest/output/etl/parquet/molecule", timestamp:1647831895},{"dataset":"Adverse Events", "description":"Significant adverse events for drug molecules", "source":"ftp://ftp.ebi.ac.uk/pub/databases/opentargets/platform/latest/output/etl/parquet/fda/significantAdverseDrugReactions", "timestamp":1647831895}]}
```
Comment (Michael9094) [inline springdb/data:1]: I'm not sure why this file is here.  Should it be deleted?

File: springdb/frontend/package.json [hunk 2]
```diff
@@ -42,6 +42,7 @@
     "@mui/lab": "^5.0.0-alpha.69",
     "@mui/material": "^5.4.2",
     "@testing-library/jest-dom": "^5.16.2",
+    "cors": "^2.8.5",
```
Comment (Michael9094) [inline springdb/frontend/package.json:45]: Is this used anywhere?  Should it be removed?

File: springdb/data (hunk 1 above)
Comment (datailluminations) [reply springdb/data:1]: Yes, it should be deleted. It was for testing purposes.

File: springdb/frontend/src/sections/@dashboard/datamgmt/CheckboxParent.js [hunk 3]
```diff
@@ -0,0 +1,84 @@
+import React, { Component } from "react";
+import Checkbox from "./Checkbox";
+import Button from "@mui/material/Button";
+
+class CheckboxParent extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      //  data_array: [],
+      selectedCheckboxes: this.props.selected
```
[the remark this answers is not in the document]
Comment (datailluminations) [reply springdb/frontend/src/sections/@dashboard/datamgmt/CheckboxParent.js]: I was experimented with a few different things. WIl delete this line.

File: springdb/frontend/package.json (hunk 2 above)
Comment (datailluminations) [reply springdb/frontend/package.json:45]: I don't use it. Probably should remove it. @Yan,  please let me know if we should delete this?

Comment (iEatYanYans) [reply springdb/frontend/package.json:45]: I'm not seeing this in my package.json. If you're not using it, let's delete

**Pull request merged**